runtime/queries/yaml/injections.scm

```diff
@@ -1,2 +1,91 @@
 ((comment) @injection.content
  (#set! injection.language "comment"))
+
+; The remaining code in this file incorporates work covered by the following
+; copyright and permission notice:
+;
+;   Copyright 2023 the nvim-treesitter authors
+;
+;   Licensed under the Apache License, Version 2.0 (the "License");
+;   you may not use this file except in compliance with the License.
+;   You may obtain a copy of the License at
+;
+;       http://www.apache.org/licenses/LICENSE-2.0
+;
+;   Unless required by applicable law or agreed to in writing, software
+;   distributed under the License is distributed on an "AS IS" BASIS,
+;   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+;   See the License for the specific language governing permissions and
+;   limitations under the License.
+
+; See https://github.com/nvim-treesitter/nvim-treesitter/blob/master/queries/yaml/injections.scm
+
+;; Github actions ("run") / Gitlab CI ("scripts")
+(block_mapping_pair
+  key: (flow_node) @_run (#any-of? @_run "run" "script" "before_script" "after_script")
```

Comment:
We don't support `#any-of?`. This will need to be rewritten as a `#match?`: `(#match? @_run "^(run|script|before_script|after_script)$")`

Reply:
Note: any-of? Was added upstream to tree-sitter now

Reply:
Ah yep, once we update tree-sitter we'll have support for `#any-of?`. The commit we are currently pinned to came right before that change

Reply:
We're looking to update again when the next release of tree-sitter gets cut but we're not sure when that will be yet